workflows/port/create_port.py

```diff
@@ -51,15 +52,14 @@ class Config:
     speed = int(_product.fixed_input_value("speed"))
 
     class CreatePortForm(FormPage):
-        class Config:
-            title = product_name
+        model_config = ConfigDict(title=product_name)
 
         # organisation: OrganisationId
 
         port_settings: Label
 
         port_ims_id: free_port_selector(node_subscription_id, speed)  # type:ignore
```

Comment:
Eventually we don't want to have function calls in Form's, just types. 

So you would have something like 'PortImsId = free_port_selector(node_subscription_id, speed)'. There is some trickery involved here to make mypy happy, unless you can live with loads of type ignores.

In some places in surf orchestrator we are already introducing this new approach, but maybe keep it here for now untill that has fully crystalized.